src/main/resources/se/lth/base/server/database/schema.sql

```diff
@@ -1,17 +1,25 @@
 -- This is the schema file that the database is initialized with. It is specific to the H2 SQL dialect.
```

Comment:
You need to figure out why this crashes as well, something is wrong with the SQL